tpm.c

```diff
@@ -209,7 +209,7 @@ EFI_STATUS tpm_log_event(EFI_PHYSICAL_ADDRESS buf, UINTN size, UINT8 pcr,
 			 const CHAR8 *description)
 {
 	return tpm_log_event_raw(buf, size, pcr, description,
-				 strlen(description) + 1, 0xd, NULL);
+				 strlen(description) + 1, EV_IPL, NULL);
```

Comment:
If I had to do it over again, I wonder if I would choose EV_EFI_VARIABLE_DRIVER_CONFIG

Reply:
That does seem better in some ways, but I don't see any good way to switch event types without breaking any code that's actually relying on the log.